test/spec/pages/source_page.rb

```diff
@@ -20,4 +20,10 @@ def title
   def avatar
     element('.source__avatar')
   end
+
+  def delete_annotation
```

Comment:
At least in the PR, you are not using this method anywhere. Please delete it if not needed.